temba_client/v2/__init__.py

```diff
@@ -131,7 +131,16 @@ def get_classifiers(self, uuid=None):
         params = self._build_params(uuid=uuid)
         return self._get_query("classifiers", params, Classifier)
 
-    def get_contacts(self, uuid=None, urn=None, group=None, deleted=None, before=None, after=None, reverse=None):
```

Comment:
did you maybe accidentally format this as line-length 80? these lines don't look so long

Reply:
updated by re-applying the latest config